### PR TITLE
Enabling testing on weston

### PR DESCRIPTION
**Pull request by yan12125**
By skipping X11-only plugins

### CODE REVIEW

File: panel/plugin.cpp [hunk 1]
```diff
@@ -259,7 +259,7 @@ bool Plugin::loadLib(ILXQtPanelPluginLibrary const * pluginLib)
     mPlugin = pluginLib->instance(startupInfo);
     if (!mPlugin)
     {
-        qWarning() << QString("Can't load plugin \"%1\". Plugin can't build ILXQtPanelPlugin.").arg(mPluginLoader->fileName());
```
Comment (yan12125) [inline panel/plugin.cpp]: ```mPluginLoader``` is nullptr in static plugins, which causes segmentation faults

**Comment by palinek** [comment]
GTM

**Comment by agaida** [comment]
GTM

**Comment by technic** [comment]
Hi, desktop list plugin is also X11 only when I've examined it last time.

**Comment by agaida** [comment]
@technic  - if so, reevalute and file a new bug

**Comment by tsujan** [comment]
> Hi, desktop list plugin is also X11 only when I've examined it last time.

I'm not sure but think that virtual desktop handling is tricky with wayland. For two of my apps, I need to know if there's a generic way *(working under all wayland DE's)* to handle virtual desktops but everything has seemed extremely vague so far. If there isn't such a way, it'll be very unfortunate, IMO.

Any link to a comprehensive doc would be really appreciated -- it would also be helpful for LXQt, of course.

**Pull request merged**